Add examples for HCP Packer (#76)
hashicorp/setup-packer: README.md

- `version` -  The version of `packer` that was installed.

## Integrating with HCP Packer

> To integrate with HCP Packer simply add your client ID and client secret as environment variables to the Packer build call, we reccomend storing these in GitHub secrets and not in plain text.  See the [HCP Packer Getting Started tutorial](https://developer.hashicorp.com/packer/tutorials/hcp-get-started/hcp-push-image-metadata)

Reviewer: ```suggestion
To integrate with HCP Packer, add your HCP Client ID and HCP Client secret as environment variables to the Packer build call.

We recommend storing these in [GitHub Actions Secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) (as opposed to plain-text). See the [HCP Packer Getting Started tutorial](https://developer.hashicorp.com/packer/tutorials/hcp-get-started/hcp-push-image-metadata)
```

```
name: hcp-packer

on:
  - push

jobs:

>   packer:

Reviewer: ```suggestion
hcp-packer
```

- `version` -  The version of `packer` that was installed.

## Integrating with HCP Packer
To integrate with HCP Packer, add your HCP Client ID and HCP Client secret as environment variables to the Packer build call.

> We add an HCP_PACKER_BUILD_FINGERPRINT in this example that is based on the workflow run ID, that way it is always unique

Reviewer 2: ```suggestion
We add an `HCP_PACKER_BUILD_FINGERPRINT` in this example that is based on the workflow run ID, that way it is always unique
```

We recommend storing these in [GitHub Actions Secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) (as opposed to plain-text). See the [HCP Packer Getting Started tutorial](https://developer.hashicorp.com/packer/tutorials/hcp-get-started/hcp-push-image-metadata)

```
name: hcp-packer

on:
  - push

jobs:
hcp-packer
    runs-on: ubuntu-latest
    name: Run Packer
    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Setup `packer`
        uses: hashicorp/setup-packer@main
        id: setup
        with:
          version: "latest"

      - name: Run `packer init`
        id: init
        run: "packer init ./image.pkr.hcl"

      - name: Run `packer validate`
        id: validate
        run: "packer validate ./image.pkr.hcl"

      - name: Build Artifact

>         run: packer build -color=false -on-error=abort packer.pkr.hcl

Reviewer 2: ```suggestion
        run: packer build -color=false -on-error=abort image.pkr.hcl
```
